docs/plugins/sponsorblock.md

```diff
@@ -0,0 +1,16 @@
+# SponsorBlock
+
+## What is SponsorBlock?
+SponsorBlock is an third party opensource extension and an API powered by community that skips automatically annoying segments in YouTube videos, like **paid sponsors**, **interaction reminders** ("Subscribe to my channel"), **resumes of previous videos**, **non-music parts** in music videos and more. It even has an option to directly go to the most important part of a video (Highlighted segment). You can customize which types of segments you want to be skipped and which you want to watch. This is integrated in **VueTube** as a plugin.
+
+*Fun fact: SponsorBlock has saved 612 years from people's time with more than 500 million skips.*
+
+## How does it work?
+Users can submit segments in videos and they go to the SponsorBlock database using a privacy preserving query system. The extension fetch segments of a video when you open it and skips them. Also you can upvote or downvote another users segments, so SponsorBlock recognizes which segments are fake or troll and ignores them. You can check more of how does it work in its [About page](https://sponsor.ajay.app/about).
```

Comment:
- reword: **which is sent to** the SponsorBlock database
- reword: you can **also** upvote or downvote
- reword: another **user's** segments so that SponsorBlock can recognize which segments are **incorrect** and ignores them
- reword: You can **read more about how** it works **through the** About Page